server/services/interventionsService.ts

```diff
@@ -202,15 +202,6 @@ export default class InterventionsService {
     })) as DraftReferral[]
   }
 
-  async getServiceProvider(token: string, id: string): Promise<ServiceProvider> {
```

Comment:
If you search for `stubGetServiceProvider`, there are some other stubs that need to be removed for this call, I think.

Reply:
Ah, well remembered. Those are gone now